webpage/jupyter_notebook_jekyll_theme/_sass/_masthead.scss

```diff
@@ -10,7 +10,7 @@
   -webkit-animation-delay: 0.15s;
           animation-delay: 0.15s;
   z-index: 20;
-  background-color: $masthead-background-color;
+  background-color: #5E5C57;
```

Comment:
I'm surprised you didn't change this in https://github.com/BiG-CZ/notebook_data_demo/blob/master/webpage/jupyter_notebook_jekyll_theme/_sass/_variables.scss#L64